base_exception/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,15 @@
+- Raphaël Valyi \<\<<raphael.valyi@akretion.com>\>\>
```

Comment:
@sbidoul seems that this conversion to MD is not correct, putting extra `<`.

Reply:
Is this entire thing just hanging on correcting this, and if so can I fix it?